central_v2/src/components/SearchAndFilter.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import useMediaQuery from '@mui/material/useMediaQuery';
+import { useTranslation } from 'react-i18next';
+import { useTheme, styled } from '@mui/material/styles';
+import { Typography, Box, Button } from '@mui/material';
+import StyledGameCard from './GameCard';
+import GameCardCarousal from './CardCarousal';
+import {ScreenSize } from '../lib/HostModels';
+import PaginationContainerStyled from '../lib/PaginationContainerStyled';
+import SearchBar from './SearchBar';
+import Filter from '../images/filter.svg';
+
+interface SearchAndFilterProps {
+  screenSize?: ScreenSize;
+}
+
+const SearchAndFilterContainer = styled(Box)(( { screenSize }: SearchAndFilterProps ) => ({
+  height: screenSize === ScreenSize.SMALL ? '70px' : '88px',
+  width: '100%', 
+  display: 'flex', 
+  justifyContent: 'center',
+  alignItems: 'center', 
+  backgroundColor: '#02215F' 
+}));
+
+const PrimaryButton2 = styled(Button)(( { screenSize }: SearchAndFilterProps ) => ({
+    width: screenSize === ScreenSize.SMALL ? 'auto' :'115px',
+    minWidth: '44px',
+    height: '38px',
```

Comment:
We're following Figma for these kinds of hardcoded values right? I ask here because this one doesn't seem to follow our 4px grid

Reply:
@drewjhart confirm sizing with design